1904-11-10.xml

```diff
@@ -0,0 +1,6722 @@
+                            <cell><measure unit="pt">127</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">132</measure></cell>
+                        </row>
+                        <row>
+                            <cell>,, Shami</cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">—</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">—</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Beans, Tugari</cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">108</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">112</measure></cell>
+                        </row>
+                        <row>
+                            <cell>,, Zawati</cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">115</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">116</measure></cell>
+                        </row>
+                        <row>
+                            <cell>,, Old</cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">—</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">—</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Lentils, Tugari</cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">108</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">112</measure></cell>
+                        </row>
+                        <row>
+                            <cell>,, Zawati</cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">122</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">128</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Barley, Tugari</cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">74</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">75</measure></cell>
+                        </row>
+                        <row>
+                            <cell>,, Zawati</cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">78</measure></cell>
+          
```

Comment:
make colSpan an attribute of the div tag in the previous line, rather than a tag all its own